.github/workflows/check_cpln_links.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Links
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  check_cpln_links:
+    runs-on: ubuntu-latest
+    name: Check Links
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
Let's use v3: `actions/checkout@v3`

There's actually an even newer version (v4), but let's at least keep it consistent with the other actions.

Reply:
Oops, good catch. Done in 7e02edf.